piker/brokers/kraken.py

```diff
@@ -129,6 +151,41 @@ class OHLC:
     ticks: List[Any] = field(default_factory=list)
 
 
+def get_config() -> dict[str, Any]:
+
+    conf, path = config.load()
+
+    section = conf.get('kraken')
+
+    if section is None:
+        log.warning(f'No config section found for kraken in {path}')
```

Comment:
so what happens if there is no section?
have you tested this case?

Reply:
I have not and will get on it

Reply:
lol just tried it. it's worse case: just halts and blocks 😂

Reply:
should I do something about it in this PR?

Reply:
hang is fixed, but need to figure out how to boot up paper